### PR TITLE
add installed.packages() step

### PR DESCRIPTION
**Pull request by dgrassellyb**
(Just adding `installed.packages()` step on R CMD check job with docker)

### CODE REVIEW

File: .github/workflows/check-using-docker.yaml [hunk 1]
```diff
@@ -1,55 +1,62 @@
...
+            echo "Let's run R CMD check..."
+            R CMD check --no-manual --no-build-vignettes ${{ steps.package-name.outputs.pkgbuild }}
+          shell: bash
+
+        # Print dependencies
+        - name: Print deps
+          run: |
+            echo "Let's print installed dependencies :"
+            installed.packages()
+          shell: bash
```
Comment (walkowif) [inline .github/workflows/check-using-docker.yaml]: I think this should run in `Rscript` shell, right?
```suggestion
          run: |
            cat("Let's print installed dependencies:\n")
            installed.packages()
          shell: Rscript {0}
```

Comment (dgrassellyb) [reply .github/workflows/check-using-docker.yaml]: Yes definitely .. thanks for fixing this :)

File: .github/workflows/check-using-docker.yaml [hunk 2]
```diff
@@ -1,55 +1,62 @@
...
+            echo "Let's run R CMD check..."
+            R CMD check --no-manual --no-build-vignettes ${{ steps.package-name.outputs.pkgbuild }}
+          shell: bash
+
+        # Print dependencies
+        - name: Print deps
+          run: |
+            cat("Let's print installed dependencies:\n")
+            installed.packages()
+          shell: Rscript {0}
```
Comment (pawelru) [inline .github/workflows/check-using-docker.yaml]: Maybe put this before `package-name` step? In the previous exercises we put such debug print before R CMD BUILD and R CMD CHECK. That would be more aligned between exercises.

Comment (dgrassellyb) [reply .github/workflows/check-using-docker.yaml]: Yes sure, I'll move it !

Comment (dgrassellyb) [reply .github/workflows/check-using-docker.yaml]: (done)

**Review by pawelru — APPROVED** [review]
👍

**Review by walkowif — APPROVED** [review]

**Pull request merged**